src/views/Login/Login.jsx

```diff
@@ -0,0 +1,65 @@
+import { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { useAuth } from '../../hooks/useAuth';
+import { signInUser, signUpUser } from '../../services/user';
+
+export function Login() {
+  const { setCurrentUser } = useAuth();
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [error, setError] = useState('');
+  const history = useHistory();
+  const [type, setType] = useState('signIn');
+
+  const handleSignIn = async (e) => {
+    e.preventDefault();
+    try {
+      const resp = await signInUser({ email, password });
+      setCurrentUser(resp);
+      history.push('/');
+    } catch (error) {
+      setError(e.message);
+    }
+  };
+
+  const handleSignUp = async (e) => {
```

Comment:
When I tried to sign in with a current user nothing happened, adding a user message here notifying them the user already exists would be a better experience.